packages/TelegramClientTests-UI.package/TCTUEmojiTests.class/instance/testEmojiExists.st

```diff
@@ -0,0 +1,7 @@
+testing
+testEmojiExists
+
+	| result |
+	
+	result := TCUEmojiHelper emojiExists: '1F3F3-200D-1F308'.
+	self assert: true equals: result.
```

Comment:
What is the point of this test? Proof of existence by example?

Reply:
Exactly, this test is there to test if the `emojiExists`-method functions correctly.
This test will actually be important, if we decide on using a faster data structure for querying emoji.

Extended the test by a invalid emoji-reference.

Reply:
Alright 👍🏻